.github/ISSUE_TEMPLATE.md

```diff
@@ -3,16 +3,32 @@
 
 ### Bug report
 
+**Matplotlib version**
+<!--Please specify your platform and versions of the relevant libraries you are using:-->
+  * Operating system:
+  * Matplotlib version:
+  * Matplotlib backend (`print(matplotlib.get_backend())`):
+  * Python version:
+  * Jupyter version (if applicable):
+  * Other libraries:
+
+<!--Please tell us how you installed matplotlib and python e.g., from source, pip, conda-->
+<!--If you installed from conda, please specify which channel you used if not the default-->
+
+
 **Bug summary**
 
 <!--A short 1-2 sentences that succinctly describes the bug-->
 
+
 **Code for reproduction**
 
 <!--A minimum code snippet required to reproduce the bug.
 Please make sure to minimize the number of dependencies required, and provide
 any necessary plotted data.
-Avoid using threads, as Matplotlib is (explicitly) not thread-safe.-->
+
+Please paste as text, not as a screenshot.
```

Comment:
Does this really happen that often that we have to mention it?

I assume if somebody just wanted to posts a screenshot, the example is not minimal and/or not working anyway. Even if that was  copy-pasted we would have to ask for improvement. - Long way of saying, I don't think this will help with better reports/less reviewer burden, so we can as well omit it.

I you really want to it, would it be an option to add it to the previous paragraph?
~~~
<!--A minimum code snippet required to reproduce the bug (not as screenshot).
~~~

Reply:
Someone was arguing with me about this on discourse over the weekend.